src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -1360,23 +1360,41 @@ def ngens(self):
         """
         return 1
 
-    def random_element(self, degree=(-1,2), *args, **kwds):
+    def random_element(self, degree=(-1, 2), monic=False, _allow_zero=False, *args, **kwds):
         r"""
-        Return a random polynomial of given degree or with given degree bounds.
+        Return a random polynomial of given degree (bounds).
 
         INPUT:
 
         -  ``degree`` - optional integer for fixing the degree
            or a tuple of minimum and maximum degrees. By default set to
            ``(-1,2)``.
 
+        -  ``monic`` - optional boolean to indicate whether the sampled
+           polynomial should be monic, or not. If this is set, `0` is not a
+           possible output of the method. If monic and `0` are both needed,
+           one should use :meth:`random_monic_or_zero_element`.
+
+        - ``_allow_zero`` - optional boolean to indicate whether zero should be
+          included. If ``monic`` is not set, or the degree bound does not
+          include `-1`, then this argument does nothing. Otherwise, the
+          algorithm returns either monic or `0` polynomials (with uniform
+          probability over the range of output). This is for internal use, users
+          should use :meth:`random_monic_or_zero_element`.
+
         -  ``*args, **kwds`` - Passed on to the ``random_element`` method for
-           the base ring
+           the base ring.
```

Comment:
```suggestion
           the base ring
```
Revert; this is against our doc style.